BookReaderDemo/demo-ia-plato.html

```diff
@@ -24,12 +24,15 @@
     <link rel="stylesheet" href="../BookReader/BookReader.css"/>
 
     <!-- plugins needed for archive.org -->
-    <script src="../BookReader/plugins/plugin.url.js"></script>
-    <script src="../BookReader/plugins/plugin.resume.js"></script>
+    <script src="../BookReader/plugins/plugin.mobile_nav.js"></script>
+    <script src="../BookReader/plugins/plugin.search.js"></script>
     <script src="../BookReader/plugins/plugin.chapters.js"></script>
     <script src="../BookReader/plugins/plugin.tts.js"></script>
-    <script src="../BookReader/plugins/plugin.search.js"></script>
-
+    <script src="../BookReader/plugins/plugin.url.js"></script>
+    <script src="../BookReader/plugins/plugin.autoplay.js"></script>
+    <script src="../BookReader/plugins/plugin.resume.js"></script>
+    <script src="../BookReader/plugins/plugin.archive_analytics.js"></script>
+    
     <link rel="stylesheet" href="BookReaderDemo.css"/>
 </head>
 <body>
```

Comment:
Suggestion: make a note in the body somewhere that this BookReader is loaded like the one we use in archive.org - and any updates to archive.org should be reflected here

Reply:
done